documentation/sphinx/source/api-c.rst

```diff
@@ -426,6 +426,25 @@ An |database-blurb1| Modifications to a database are performed via transactions.
    ``*out_transaction``
       Set to point to the newly created :type:`FDBTransaction`.
 
+.. function:: FDBFuture* fdb_database_reboot_worker(FDBDatabase* database, uint8_t const* address, int address_length, fdb_bool_t check, int duration)
+
+   Reboot the specified process in the database.
+
+   |future-return0| a :type:`int64_t` represents whether the reboot request is successful or not. In particular, 1 means success and 0 means failure. |future-return1| call :func:`fdb_future_get_int64()` to extract the result, |future-return2|
```

Comment:
To me a successful request sounds like we received an acknowledgement that the request's intended effect has taken place. That's not the case here. Can we make this something like

    In particular, 1 means request sent and 0 means failure (e.g. the storage with the specified address does not exist)

I also think it's worth considering having this be a Future<Void> and it can fail with an error if the storage doesn't exist

Reply:
Make sense. Done.